packages/core/src/browser/preferences/preference-provider.ts

```diff
@@ -5,11 +5,28 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
+import { injectable } from 'inversify';
 import { Disposable, Event } from '../../common';
 
 export const PreferenceProvider = Symbol('PreferenceProvider');
 
 export interface PreferenceProvider extends Disposable {
     getPreferences(): { [key: string]: any };
+    setPreference(key: string, value: string): Promise<void>
```

Comment:
`value` should be `any` too, to be symmetrical to the access.

Reply:
done